modules/dns-helper/dns_local_server.go

```diff
@@ -0,0 +1,112 @@
+package dns_helper
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"strings"
+	"time"
+
+	"github.com/miekg/dns"
+)
+
+var testDomain = "gruntwork.io"
+
+// DNSData type
+type DNSData map[DNSQuery][]DNSAnswer
+
+func setupTestDNSServers() (ns1, ns2 string, dnsData1, dnsData2 *DNSData) {
+	ns1, mux1 := runTestDNSServer("0")
+	ns2, mux2 := runTestDNSServer("0")
+
+	q := DNSQuery{"NS", testDomain}
+	dnsData1 = &DNSData{q: DNSAnswers{{"NS", ns1}, {"NS", ns2}}}
+	dnsData2 = &DNSData{q: DNSAnswers{{"NS", ns1}, {"NS", ns2}}}
+
+	mux1.HandleFunc(testDomain+".", func(w dns.ResponseWriter, r *dns.Msg) { stdDNSHandler(w, r, dnsData1, false) })
+	mux2.HandleFunc(testDomain+".", func(w dns.ResponseWriter, r *dns.Msg) { stdDNSHandler(w, r, dnsData2, true) })
+
+	return
```

Comment:
Style wise, we prefer explicit return statements vs implicit. Can you switch to explicitly returning each val here? This applies to all functions in this file.

Reply:
Ensured all return statements are explicit.